test/AppKernel.php

```diff
@@ -11,12 +11,18 @@ class AppKernel extends Kernel
     private $configuration;
     private $enabledPlugins;
 
-    public function __construct(array $configuration, array $enabledPlugins)
+    private $cacheDir;
+    private $logsDir;
+
+    public function __construct(array $configuration, array $enabledPlugins, $uniqueIdentifier, $cacheDir, $logsDir)
     {
-        parent::__construct('test', true);
+        parent::__construct('test' . $uniqueIdentifier, true);
```

Comment:
Ah, sorry about that. I was too fast. And sorry again, I didn't think hard enough about that. Since we're actually reconfiguring the AppKernel itself to test different app setups, we need this separation, right?